docs/reference/operators/data/filter.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Filter
+---
+
+The Filter operator filters events in user space. Since the filtering in user
+space is slower, it's preferred to use in-ebpf filtering options when possible,
+as provided by other operators like [LocalManager](./localmanager.md) and
+[KubeManager](./kubemanager.md) or specific
+[parameters](../../gadget-devel/parameters.md) provided by each gadget.
+
+## Instance Parameters
+
+### `filter`
+
+This parameter allows you to filter events based on specific field values
+provided by the gadget. This is particularly useful for narrowing down the
+output to entries that meet certain criteria.
+
+The filter syntax supports the following operations:
+
+```bash
+- `columnName==value`: Matches if the content of `columnName` equals exactly `value`.
```

Comment:
Please apply changes in https://github.com/inspektor-gadget/inspektor-gadget/pull/3416.

Reply:
ops, thanks !